### PR TITLE
Add indexer column to home

### PR DESCRIPTION
**Pull request by fernandog**
@p0psicles 

image

TODO:
- [x] Add to "poster" layout (where exactly to add it?)
- [x] Sort and filter by indexer in layout "simple" is not working @p0psicles @OmgImAlexis

### CODE REVIEW

**Comment by duramato** [comment]
Poster  view shouldn't have it as you can sort by columns in it, or if you really want it, add it next to the network icon.

**Comment by fernandog** [comment]
poster view doesn't have indexer column

**Comment by duramato** [comment]
Exactly, it doesn't even have columns re: `TODO:

 Add to "poster" layout (where exactly to add it?)`

**Comment by fernandog** [comment]
can anyone test this? all layouts are working. just need someone else to check it

**Comment by p0psicles** [comment]
Can you rebase it on my responsive pr? I already had to fix allot of conflict.

**Comment by fernandog** [comment]
@p0psicles @OmgImAlexis this last commit worked in my IDE but its not working in my production. any idea whats wrong?

```javascript
                indexer: function(itemElem) {
                    var indexer = $(itemElem).attr('data-indexer');
                    return (indexer.length && parseInt(indexer, 10)) || Number.NEGATIVE_INFINITY;
```
image

**Comment by p0psicles** [comment]
Ooh never mind. I just noticed you didn't touch displayShow. No  need to rebase then

**Comment by fernandog** [comment]
@p0psicles can we merge this before the other responsive PR?

File: static/js/home/index.js [hunk 1]
```diff
@@ -93,25 +93,30 @@ MEDUSA.home.index = function() {
...
-            5: function(node) { return $(node).find('span:first').text(); }, // eslint-disable-line brace-style
-            6: function(node) { return $(node).data('show-size'); }, // eslint-disable-line brace-style
-            7: function(node) { return $(node).find('img').attr('alt'); } // eslint-disable-line brace-style
-        },
+        textExtraction: (function() {
+            return {
+                0: function(node) { return $(node).find('time').attr('datetime'); }, // eslint-disable-line brace-style
+                1: function(node) { return $(node).find('time').attr('datetime'); }, // eslint-disable-line brace-style
+                3: function(node) { return $(node).find('span').prop('title').toLowerCase(); }, // eslint-disable-line brace-style
+                4: function(node) { return $(node).find('a[data-indexer-name]').attr('data-indexer-name'); }, // eslint-disable-line brace-style
```
Comment (p0psicles) [inline static/js/home/index.js]: I've added this but probably not even needed.

File: static/js/home/index.js [hunk 2]
```diff
@@ -93,25 +93,30 @@ MEDUSA.home.index = function() {
...
         widgets: ['saveSort', 'zebra', 'stickyHeaders', 'filter', 'columnSelector'],
         headers: {
             0: {sorter: 'realISODate'},
             1: {sorter: 'realISODate'},
             2: {sorter: 'loadingNames'},
-            4: {sorter: 'quality'},
-            5: {sorter: 'eps'},
-            6: {sorter: 'digit'},
-            7: {filter: 'parsed'}
+            4: {sorter: 'text'},
```
Comment (p0psicles) [inline static/js/home/index.js]: This was the issue. So I know for sure you have to add this.

**Pull request merged**